system/system_error_monitor/src/system_error_monitor_core.cpp

```diff
@@ -28,6 +28,34 @@
 
 namespace
 {
+enum class DebugLevel { DEBUG = 0, INFO, WARN, ERROR };
```

Comment:
Seeing https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#enum8-specify-enumerator-values-only-when-necessary,

You have to remove `= 0`,

```suggestion
enum class DebugLevel { DEBUG, INFO, WARN, ERROR };
```

or add some comments?

```suggestion
enum class DebugLevel { DEBUG = 0, INFO, WARN, ERROR }; // To ...
```

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/510/commits/142b5a553a992dc53d0db0231db87439c646f0d9
I don't understand the second suggestion, but Doen

Reply:
I meant, for example `To be the same as rclcpp's macros`

Reply:
Just go it.